package.json

```
@@ -76,6 +79,7 @@
     "eslint": "^6.3.0",
     "eslint-config-standard": "^14.1.1",
     "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jsdoc": "^25.4.2",
```

```suggestion
    "eslint-plugin-jsdoc": "^30.4.2",
```

upgrade to at least `v29.0.0` to [support type intersections (`X & Y`)](https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v29.0.0)